File: responses/SDXU35-KKU/sum_of_certain_multiples.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

const sum = (val, next) => val + next;

function solution(x) {
    let numbers = [];

    for (let i=1; i < x; i++) {
        if (i % 3 === 0) {
            if (i % 5 !== 0) {
                numbers.push(i);
            }
        } else if (i % 5 === 0) {
            numbers.push(i);
        }
    }

    return numbers.reduce(sum);
}

File: responses/UZRFMM-MY2/sum_of_certain_multiples.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(x):
    count = x
    sum = 0
    while count > 0:
        count -= 1
        if (count % 3 == 0) or (count % 5 == 0):
            sum += count
    return sum